inc/blocks/ab-test/register.php

```diff
@@ -113,7 +113,7 @@ function render_block( array $attributes, ?string $inner_content = '' ) : string
 		$winner = $results['winner'] ?? false;
 	}
 
-	if ( current_user_can( 'edit_post', $post->post_parent ?? 0 ) && ( is_preview() || isset( $_GET['set_test'] ) ) ) {
+	if ( current_user_can( 'edit_post', $post->post_parent ?? 0 ) && ( is_preview() || is_customize_preview() ) ) {
```

Comment:
I'm thinking about it now and if we only preview on `is_preview()` or `is_customize_preview()` then the user will have appropriate privileges so we can drop the `current_user_can()` part - the XB post won't always have a parent post in the case of those added as widgets.
```suggestion
	if ( is_preview() || is_customize_preview() ) {
```

Reply:
As much as I had thought about dropping that... 

>Any visitor can add ?preview=true, ?preview=1, or ?preview=not_false to the URL, and is_preview() will return true.

I realise it's [mentioned in the comments](https://developer.wordpress.org/reference/functions/is_preview/#user-contributed-notes) and not the official docs so it may be worthwhile keeping it in place?

Reply:
Oh right, hmmm. Thing is if the XB has no parent post that'll never return true except for super admins. Maybe just change it back to the primitive capability `current_user_can( 'edit_posts' )` then.